src/decision_table_utils.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+#ifndef CATA_SRC_DECISION_TABLE_UTILS_H
+#define CATA_SRC_DECISION_TABLE_UTILS_H
+
+#include <map>
+
+/**
+ * @brief Creates a lookup lambda using a decision table.
+ *
+ * for example, it could be used to create a lambda that returns a color based on a stat value:
+ * { ~ 25: magenta, 25 ~ 50: red, 50 ~ 75: yellow, 75 ~ 100: green }
+ *
+ * @param table keys are the lower bound, inclusive. passed as an rvalue reference to avoid unnecessary copying.
```

Comment:
```suggestion
 * @param table keys are the upper bound, inclusive. passed as an rvalue reference to avoid unnecessary copying.
```
forgot to change that line after converting internal logic from lower_bound to upper_bound.